tests/main/snap-env/task.yaml

```diff
@@ -1,63 +1,85 @@
 summary: inspect all the set environment variables prefixed with SNAP_ and XDG_
 
+environment:
+    NAME: test-snapd-tools
```

Comment:
This is slightly unusal AFAICT. In most tests that use environment we have a "/description" for all uses. Mixing "NAME" and "NAME/paralle" is slightly unusual. Maybe "NAME/normal" and "NAME/parallel" (same for insance_key)?